### ISSUE TITLE
Uploaded file name not included in data export

### ISSUE THREAD

**Issue by stevefloat** [verbb/formie#476]
Hi there

I have a form set up containing a File Upload field.

This all works well and the file is pushed up to our Amazon S3 bucket. The form data can be previewed in 'Submissions' and can confirm the uploaded file exists within the 'Assets' library.

When you choose to export your list of submissions as a CSV file via your 'Export' option, there is a column for the File Upload field in the CSV file but there is no content - the fields are blank.

It would be very useful if this field in the CSV file contained the uploaded file's name - that which is generated dynamically on submission.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.27
- Craft version: 3.5.19.1
- Multi-site: Yes

**Comment by engram-design** [comment]
I believe this might come down to the `Assets in this volume have public URLs` setting for the volume. I've added a fallback if a URL can't be generated, to use the filename - as this can happen for local volumes as well.

Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.4",
  "...": "..."
}
```

Then run `composer update`.
